### PR TITLE
FAQ: add common sense FAQs

### PR DESCRIPTION
**Pull request by moneromooo-monero**
Some people apparenly lack such common sense, but get angry at not
being told in LARGE LETTERS than having common sense is better than not.

### CODE REVIEW

**Comment by el00ruobuob** [comment]
Hi @moneromooo-monero, could you add those two new questions and answers in other yml language files too? It'll help the translation team to get those translated.

File: _i18n/en.yml [hunk 1]
```diff
@@ -293,6 +293,10 @@ faq:
...
   a12-3: As long as the encrypted output amounts you create is equal to the sum of the inputs that are being spent (which include an output for the recipient and a change output back to yourself and the unencrypted transaction fee), then you have a legitimate transaction and know no Monero is being created out of thin air. Pedersen commitments mean that the sums can be verified as being equal, but the Monero value of each of the sums and the Monero value of the inputs and outputs individually are undeterminable.
+  q13: Is Monero magic so I can do dumb things and I'll be anonymous ?
+  a13: Monero is not magic. Magic does not exist. If you use Monero but give your name and address to another party, the other party will not magically forget your name and address. If you give out your secret keys, others will know what you've done. If you get pwned, others will be able to keylog you. If you use a weak password, others will be able to brute force your keys file. If you backup your seed in the cloud, you'll be poorer soon.
```
[the remark this answers is not in the document]
Comment (moneromooo-monero) [reply _i18n/en.yml]: pwned. I guess it could be replaced with "compromised" if not widely known.

**Comment by moneromooo-monero** [comment]
Sorry, I don't speak those other languages.

**Comment by el00ruobuob** [comment]
It's not about translating them yourself, just keeping the yml files with all the variables so that the translators knows they have something new to translate. But if you don't have time, I can do it in #837

File: _i18n/en.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply _i18n/en.yml]: It wasn't obvious to me, but I may not be a reference.

**Comment by moneromooo-monero** [comment]
Like this ?

**Comment by moneromooo-monero** [comment]
I also replaced pwned with compromised and rephrased some of the sarcastic wording :)

**Pull request merged**